src/number.rs

```diff
@@ -31,6 +31,18 @@ enum N {
     Float(f64),
 }
 
+/// Describes the kind of number we're dealing with
+/// This is a hint about how the number is stored internally
```

Comment:
This needs punctuation and a newline in between, or else rustdoc sees it as one sentence.

---

![selection_071](https://user-images.githubusercontent.com/1940490/31508426-e0540f78-af32-11e7-8653-e395c0e9fb9e.png)